tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  purge: ['./src/**/*.{js,jsx,ts,tsx}'],
```

Comment:
The `purge` feature is to remove unused style by scanning on the configured path, more info on how it work can be found [here](https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html)